tutorials/hoon/conditionals.md

```diff
@@ -11,13 +11,12 @@ In this lesson, we will write a generator that takes an integer and checks if it
 :-  %say
 |=  [* [n=@ud ~] ~]
 :-  %noun
-?:  ?&  =(0 (mod n 2))
-        ?&  (gte n 1)
-            ?!  (gth n 100)
-        ==
-    ==
-    %.y
-%.n
+^-  ?
+?&
+    =(0 (mod n 2))
```

Comment:
I put `=(0 (mod n 2))` on a separate line from `?&` for the sake of the following explanation, but this is bad practice (I think) so I'm unsure about this choice and whether its worth making the explanation slightly more confusing by using `?&  =(0 (mod n 2))` instead